postgrest.cabal

```diff
@@ -44,8 +44,8 @@ executable postgrest
                      , http-types
                      , interpolatedstring-perl6
                      , jwt
-                     , microlens >= 0.4.2 && < 0.5
-                     , microlens-aeson >= 2.1.1 && < 2.2
+                     , lens >=3.8 && < 5.0
```

Comment:
That's a dependency dragged in by swagger2:
https://github.com/haskell-servant/servant-swagger/issues/39